etna/core/models/settings.py

```diff
@@ -12,15 +13,23 @@
 
 @register_setting(icon="list-ul")
 class SiteSettings(BaseSetting, ClusterableModel):
-    panels = [
-        InlinePanel(
-            "main_menu_items_rel",
-            heading="main menu items",
-            label="item",
-            min_num=1,
-            max_num=6,
-        ),
-    ]
+    stand_first = models.CharField(max_length=200, blank=False, null=True)
```

Comment:
As per guidance in the [Django docs](https://docs.djangoproject.com/en/4.0/ref/models/fields/#null)), `null=True` is best avoided on `CharField`s (also true for `URLField` and `RichTextField`).

Reply:
Also the name `stand_first` is a little ambiguous. I would suggest `beta_banner_standfirst` for this field, and add `verbose_name="standfirst"` to avoid repeating the 'beta banner' text in the field labels (we'll use a `MultiFieldPanel` to group them under a heading)